runtime/src/bank.rs

```diff
@@ -1183,7 +1189,14 @@ impl Bank {
             vote_only_bank: false,
             cost_tracker: RwLock::<CostTracker>::default(),
             sysvar_cache: RwLock::new(Vec::new()),
-        }
+            accounts_data_len: AtomicU64::default(),
+        };
+
+        let total_accounts_stats = bank.get_total_accounts_stats().unwrap();
+        bank.accounts_data_len
+            .store(total_accounts_stats.data_len as u64, Release);
```

Comment:
I was/am worried where `Bank::default_with_accounts()` is called. I've only seen it in tests, and in those cases the accounts data len here is always zero, and the time the `get_total_accounts_stats()` takes to run is 50-100 us on my MBP. let me know if this should be changed though.

Reply:
maybe we rename it to default_for_tests or something and make tests be explicit?

Reply:
Ah, I see, `Bank::default_with_accounts()` is called by `Bank::new_with_paths()`, which is called by blockstore_processor when starting from Bank 0. So I'm going to leave the bank functions named the way they are. I feel pretty good, since starting from Bank 0 should have zero accounts (or very few), and the get_total_accounts_stats to get the accounts_data_len will be fast.